packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -284,75 +288,99 @@ export const getSyncs = async (nangoConnection: Connection): Promise<(Sync & { s
         );
 
     const result = await q;
-    const syncsWithSchedule = result.map(async (sync) => {
-        const { schedule_id } = sync;
-        const syncSchedule = await syncClient?.describeSchedule(schedule_id);
 
-        if (syncSchedule) {
-            if (syncSchedule.schedule?.state?.paused && sync.schedule_status === SyncStatus.RUNNING) {
-                sync = {
-                    ...sync,
-                    schedule_status: SyncStatus.PAUSED
-                };
-                await updateScheduleStatus(schedule_id, SyncCommand.PAUSE, null, nangoConnection.environment_id);
-                await telemetry.log(
-                    LogTypes.TEMPORAL_SCHEDULE_MISMATCH_NOT_RUNNING,
-                    'UI: Schedule is marked as paused in temporal but not in the database. The schedule has been updated in the database to be paused.',
-                    LogActionEnum.SYNC,
-                    {
-                        environmentId: String(nangoConnection.environment_id),
-                        syncName: sync.name,
-                        connectionId: nangoConnection.connection_id,
-                        providerConfigKey: nangoConnection.provider_config_key,
-                        syncId: sync.id,
-                        syncJobId: String(sync.latest_sync?.job_id),
-                        scheduleId: schedule_id,
-                        level: 'warn'
-                    },
-                    `syncId:${sync.id}`
-                );
-            } else if (!syncSchedule.schedule?.state?.paused && sync.schedule_status === SyncStatus.PAUSED) {
-                sync = {
+    const isGloballyEnabled = await featureFlags.isEnabled('orchestrator:schedule', 'global', false);
+    const isEnvEnabled = await featureFlags.isEnabled('orchestrator:schedule', `${nangoConnection.environment_id}`, false);
+    const isOrchestrator = isGloballyEnabled || isEnvEnabled;
+    if (isOrchestrator) {
+        const searchSchedulesProps = result.map((sync) => {
+            return { syncId: sync.id, environmentId: nangoConnection.environment_id };
+        });
+        const schedules = await orchestrator.searchSchedules(searchSchedulesProps);
```

Comment:
Scheduler is now called by the UI (with auto-refresh). That might put a lot of pressure on the orchestrator service if a lot of connections page stays open. We'll see if this is a problem. At least now we are doing a single request to fetch all schedules instead of one by one